spec/factories.rb

```diff
@@ -0,0 +1,22 @@
+FactoryGirl.define do
```

Comment:
You have to use one file per factory.
spec/factories/book.rb
spec/factories/user.rb
spec/factories/rent.rb

Also, to avoid repetition in Faker user the following format:

`genre { Faker::Book.genre }`